postgres/README.md

```diff
@@ -1,19 +1,119 @@
+
+- Install [Postgres](https://www.postgresql.org/download/). Uppon downloading, you should be able to successfully run the following commands:
+
+```bash
+createdb --help
+psql --help
+```
+
+- Spice is installed (see the [Getting Started](https://docs.spiceai.org/getting-started) documentation).
+
+- [Login](https://docs.spiceai.org/cli/reference/login) to Spice, since this quickstart is using [Spice.ai Data Connector](https://docs.spiceai.org/data-connectors/spiceai).
+
+```bash
+spice login
+```
+
+**Step 1.** Create a Postgres database for testing.
+
+Create an empty database `spice_demo` in local Postgres.
+
+```bash
+createdb spice_demo
+```
+
+Verify the creation of `spice_demo` by openning it with psql CLI.
+
+```bash
+psql spice_demo
+```
+
+**Step 2.** Initialize a Spice app.
+
+```bash
+spice init postgres-demo
+cd postgres-demo
+```
+
+**Step 3.** Start the Spice runtime.
+
+```bash
+spice run
+```
+
+The Spice runtime terminal will show that Spice Runtime is running.
+
+```
+Spice.ai runtime starting...
+2024-05-07T01:01:40.566270Z  INFO spiced: Metrics listening on 127.0.0.1:9000
+2024-05-07T01:01:40.566873Z  INFO runtime::flight: Spice Runtime Flight listening on 127.0.0.1:50051
+2024-05-07T01:01:40.566960Z  INFO runtime::opentelemetry: Spice Runtime OpenTelemetry listening on 127.0.0.1:50052
+2024-05-07T01:01:40.568738Z  INFO runtime::http: Spice Runtime HTTP listening on 127.0.0.1:3000
+```
+
+**Step 4.** Configure the dataset to use Postgres as data accelerator. Copy and paste the configuration below to `spicepod.yaml` in the Spice app.
+
+```yaml
+version: v1beta1
+kind: Spicepod
+name: postgres-demo
+datasets:
+  - from: spice.ai/eth.recent_blocks
+    name: eth_recent_blocks
+    acceleration:
+      enabled: true
+      refresh_mode: full
+      refresh_check_interval: 10s
+      engine: postgres
+      params:
+        pg_host: localhost
+        pg_port: 5432
+        pg_db: spice_demo
+        pg_sslmode: disable
+```
+
+Save the changes to `spicepod.yaml`. The Spice runtime terminal will show that the dataset has been loaded:
+
+```
+2024-05-07T23:56:44.094995Z  INFO runtime: Loaded dataset eth_recent_blocks
+2024-05-07T23:56:44.095902Z  INFO runtime::accelerated_table: [refresh] Loading data for dataset eth_recent_blocks
+```
 
 Follow the [quickstart guide](https://docs.spiceai.org/getting-started) to get started with the Spice.ai runtime.
 
-See the [datasets reference](https://docs.spiceai.org/reference/spicepod/datasets) for more dataset configuration options and [PostgreSQL Data Accelerator](https://docs.spiceai.org/data-accelerators/postgres) for more options on configuring PostgreSQL as a Data Accelerator.
+See the [datasets reference](https://docs.spiceai.org/reference/spicepod/datasets) for more dataset configuration options.
 
-To securely store your PostgreSQL password, see [Secret Stores](https://docs.spiceai.org/secret-stores)
+**Step 5.** Run queries against the dataset using the Spice SQL REPL.
```

Comment:
I think it would also be cool to add a step showing how to query the same table using `psql` to show it is just a table in Postgres.

Reply:
Added psql section after spice sql